website/pages/docs/advanced-topics/environment-variable-substitution.md

```diff
@@ -0,0 +1,19 @@
+# Environment variable substitution
+
+CloudQuery configuration `.yml` files support substitution of values
+from environment variables. This allows you to keep sensitive data (like passwords & tokens) or variable data (that you want to change without touching CloudQuery configuration) outside the configuration file and load them from environment variables at run-time.
+
+## Example
+
+And use it inside `gcp.yml`:
+
+```yaml
+kind: destination
+spec:
+  name: postgresql
+  tables: [""]
```

Comment:
```suggestion
kind: "destination"
spec:
  name: "postgresql"
```

Reply:
Why do we want the `"` here?

Reply:
Consistency with how the CLI generates it, it quotes all strings:
https://github.com/cloudquery/cloudquery/blob/65badc114fb39f63924c34d989d487968070ff40/cli/cmd/templates/destination.go.tpl#L1